epytope/test/TestSpacerDesign.py

```diff
@@ -15,8 +15,8 @@ class SpacerDesignTestCase(unittest.TestCase):
     """
 
     def setUp(self):
-        epis = """GHRMAWDMM
-                 VYEADDVIL""".split("\n")
+        epis = """GHRMAWDM
```

Comment:
The last 'M' of `GHRMAWDMM ` is removed and makes it an 8mer. Purpose is to check if the program throws an error when the length is not supported. Previously A26:01 with length 9 was not supported, but I added a matrix for this, so I made the peptide an 8mer such that it should throw an error because length 8 is now not supported anymore.

Reply:
I see. Perfect, thanks!